javascript-data-model/order-history.js

```diff
@@ -0,0 +1,61 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    orderType: [{
+      type: 'Book',
+      title: 'Javascript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel'
+    }],
+    shipping: [{
+      orderPlaced: 'August 8, 2020',
+      shippingStatus: 'Delivered August 8, 2020',
+      returnWindow: 'closed on Sep 7, 2020'
+    }],
+    costTotal: 31.55
+  }, {
+    orderNumber: '113-9984268-1280257',
+    orderType: [{
+      type: 'Book',
+      title: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher'
+    }],
+    shipping: [{
+      orderPlaced: 'July 19, 2020',
+      shippingStatus: 'Delivered Jul 20, 2020. Your package was delivered. It was handed directly to a resident.',
+      returnWindow: 'closed on Aug 19, 2020'
+    }],
+    costTotal: 44.53
+  }, {
+    orderNumber: '114-2875557-9059409',
+    orderType: [{
+      type: 'Electronics/Gaming',
+      title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter'
+    }],
+    shipping: [{
+      orderPlaced: 'July 3, 2020',
+      shippingStatus: 'Delivered Jul 5, 2020',
+      returnWindow: 'closed on Aug 4, 2020'
+    }],
+    costTotal: 15.98
+  }, {
+    orderNumber: '113-2883177-2648248',
+    orderType: [{
+      type: 'Electronics/Gaming',
+      title: 'GameCube Controller Adapter'
+    }, {
+      type: 'Book',
+      title: 'The Art of Sql',
+      author: 'Faroult, Stephane'
+    }],
+    shipping: [{
+      orderPlaced: 'July 3, 2020',
+      shippingStatus: 'Delivered Jul 5, 2020',
+      returnWindow: 'closed on Aug 4, 2020'
+    }],
```

Comment:
I don't think the value of this property needs to be an array, there's no possibility of multiple shipping locations for a single order.